SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -94,12 +132,12 @@ TOptional<Trace_SpanId> SpatialEventTracer::CreateSpan(const Trace_SpanId* Cause
 		return {};
 	}
 
-	if (NumCauses > 0)
+	if (Causes != nullptr && NumCauses > 0)
```

Comment:
Should this also merge with the span stack? I feel like it should

Reply:
Also some weird behavior, if there are no causes then it does merge with the span stack otherwise it doesn't